### ISSUE TITLE
Maintenance mode automatically deactivated after cache:clear

### ISSUE THREAD

**Issue by agoat** [contao/core-bundle#728]
When clearing the symfony cache with `php bin/console cache:clear -e prod` the status of the maintenance mode is also cleared. Is it possible to change this behaviour?

To stay in maintenance mode after updating and clearing the cache, would be helpfull to check, if everything works properly before open it again for the visitors.

**Comment by leofeyer** [comment]
We must not store the lock file in the Sf cache directory. Here's the default Lexik configuration:

```yaml
driver:
        options: {file_path: %kernel.root_dir%/cache/lock}
```

**Comment by Toflar** [comment]
Imho it should just go to `var/maintenance_lock` or so. :)

**Comment by aschempp** [comment]
Unfortunately the default Lexik configuration is not meant for Symfony 3, but it's correct their lock file is outside the environment cache directory. I think @Toflar's suggestion makes sense, another option might be `%kernel.cache_dir/../(maintenance_)lock`

**Comment by leofeyer** [comment]
Fixed in 421a23aed19a01f1fddb01bdfceec4f69dc1e921.
